omod-1.9/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_9/RestConstants1_9.java

```diff
@@ -15,8 +15,9 @@
  * {@link RestConstants} for 1.9 resources.
  */
 public class RestConstants1_9 {
-
-	private RestConstants1_9(){}
+	
+	private RestConstants1_9() {
+	}
```

Comment:
Is the formatting changes part of this ticket? It's very good that you want the code to look nice, but changes to files that are out of ticket range shouldn't be done.

Reply:
I didn't make changes in those modified files But here is my findings in respect to those changes:
mvn clean install  allows the automatic code formatter to format code according to the OpenMRS conventions. That’s probably why those files are modified by that process, and the committer should likely commit those changes to the main repo eventually.

Reply:
Yes, maybe you are right.